src/Service.Host/client/src/components/requisitions/components/DepartmentNominal.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import Grid from '@mui/material/Grid2';
+import { InputField, Search } from '@linn-it/linn-form-components-library';
+import PropTypes from 'prop-types';
+import itemTypes from '../../../itemTypes';
+import useSearch from '../../../hooks/useSearch';
+
+function DepartmentNominal({
+    departmentCode = null,
+    departmentDescription = null,
+    setDepartment,
+    nominalCode = null,
+    nominalDescription = null,
+    setNominal,
+    shouldRender = true
+}) {
```

Comment:
could pass a disabled prop? might need to be able to switch these on and off? Suppose you could just handle that in the two setFunctions too (i.e. only actually do something under certain conditions)

Reply:
yeah i'm assuming most of these things might be extended in various ways in the future